src/docs/content/observability/oci-healthcheck.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "OCI Health Check"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 0
+tags:
+  - Monitoring
+  - oci_healthcheck
+---
+
+{{% alert style="warning" icon="warning" %}}
+Note that this is **OPTIONAL**.  
+{{% /alert %}}
+
+## Introduction
+
+Monitors the health of IP addresses and hostnames, as measured from geographic vantage points of your choosing, using HTTP and ping probes. After configuring a health check, you can view the monitor's results. The results include the location from which the host was monitored, the availability of the endpoint, and the date and time the test was performed.
+
+## Create OCI HealthCheck
+
+Click Monitoring -> Health Checks -> Create HealthCheck
+
+![OKE Cluster Metrics](../images/create-healthcheck.png)
+
+{{% alert style="primary" icon="warning" %}}
+- Target will be IP of ingress controller
+```
```

Comment:
This code block seems indented oddly. is this be cause of the `alert` wrapper?

Reply:
yes the alert is causing the issue.. if ok, can remove the alert or lets go with as is. let me know.

Reply:
re-open if you disagree.